app/assets/javascripts/chat.coffee

```diff
@@ -99,9 +99,8 @@ class Chat
         $(@).replaceWith html+orig.slice(idx) if html
 
   norlogize_left: (x) ->
-    r = /((\d{4}-\d{2}-\d{2}) )?(\d{2}:\d{2}:\d{2})/g
```

Comment:
This regexp allowed to have only time in the innerHTML.

I've assumed that back reference "$2" will be empty if first group is not matched (as allowed by the quotation mark).

With new code, I assume to always have date and time inside the datetime attribute, because the board.created_at is formatted in ISO 8601.

But I've just read on wikipedia that this format can contains only time.
Should I set date to empty string "", because date will be null in that case ?